### ISSUE TITLE
Add Cron entry for `docker system prune`

### ISSUE THREAD

**Issue by jaydrogers** [serversideup/ansible-collection-spin#6]
# Problem
Docker systems can produce a lot of clutter

# Proposed solution
- [ ] Add a cron entry for `00 6 * * * /usr/bin/docker system prune --all --force`
- [ ] Ansible should automatically add this on `spin provision`

**Comment by luchtech** [comment]
Thanks you, @jaydrogers . This feature will be a huge time-saver when debugging what went wrong on our server. 😆

**Comment by luchtech** [comment]
I agree. The best place for this is during "spin provision" since it's outside of Docker Swarm. It also makes sense if this can be triggered right after every deployment. Whichever is simpler to implement is the better option.

**Comment by jaydrogers** [comment]
This will be fixed with this gets merged 😃

- https://github.com/serversideup/ansible-collection-spin/pull/9
